scripts/devshell/prettify

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+
+# Function to display help message
+show_help() {
+    echo "Usage: format <OPTION|FILES>"
+    echo "Format Haskell source files using fourmolu and stylish-haskell."
+    echo ""
+    echo "Options:"
+    echo "  -t, --tracked     Format all tracked Haskell (*.hs) files in the repository"
+    echo "  -s, --staged      Format all staged Haskell (*.hs) files"
+    echo "  -m, --modified    Format all modified Haskell (*.hs) files, including staged and unstaged"
+    echo "  -n, --not-staged  Format all non-staged modified Haskell (*.hs) files"
+    echo "  -h, --help        Show this help message"
+}
+
+# Function to run the formatting commands
+run_format() {
+    for file in "$@"; do
+        if [[ $file == *.hs ]]; then
+            if grep -qE '^#' "$file"; then
+              echo "$file contains CPP.  Skipping."
+            else
+              echo "Formatting: $file"
+              fourmolu -q -i "$file"
+              fourmolu -q -i "$file"
```

Comment:
Why do you execute the same command twice?

Reply:
I found instances of `fourmolu` not being idempotent, but it became idempotent after running it twice. So I did this to minimize glitches, basically